content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Dapatkan _Shell_ Untuk Masuk ke Kontainer yang Sedang Berjalan
```

Comment:
```suggestion
title: Mendapatkan Shell Untuk Masuk ke Container yang Sedang Berjalan
```

Let's don't use italic, and leave Container be it as it's an API Object.

Reply:
Ok! sorry my bad :)